dependency-reduced-pom.xml

```diff
@@ -29,7 +29,7 @@
       </plugin>
       <plugin>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>3.3.0-SNAPSHOT</version>
+        <version>3.3.0</version>
```

Comment:
Intellij couldn't find the SNAPSHOT version so I wasn't able to compile it until I changed the shade plugin version, I can rollback this to previous value.